held/app/src/main/java/com/held/activity/VerificationActivity.java

```diff
@@ -159,34 +163,39 @@ public void afterTextChanged(Editable editable) {
     }
 
     private void callVerificationApi() {
-        HeldService.getService().verifyUser(mPhoneNo, mPin, new Callback<VerificationResponse>() {
+        HeldService.getService().verifyUser(mAuth,mRegId, mPin,"", new Callback<VerificationResponse>() {
                     @Override
                     public void success(VerificationResponse verificationResponse, Response response) {
-                        if (verificationResponse.isVerified()) {
```

Comment:
isVeryfied() is method for Boolean veriable considered to be set true in response object but its not happening, and if success() is successfully executes it means response is good according to Retrofit API. so we dont need to set variable and check it that is it true or not. by the way in response we are not have boolean variable or object for verification.